content/en/core/guides/android/branding.md

```diff
@@ -219,9 +225,9 @@ Because the files generated here are signed with the same key that you are going
 
 ### 5. Release the new flavor
 
-Once you have your pull request approved in the cht-android repo, it's recommended to create an alpha version before merge it to master and do the final release.
+Make a pull request to the release branch in the CHT Android repository, once approved and before merging, it's recommended to create an alpha version to do final tests before releasing.
```

Comment:
This is now quite long. How about this instead?

1. Make a Pull Request to the release branch in the CHT Android repository.
2. Once approved it's recommended to create an alpha version to do final tests.
3. Merge the PR.
4. [Release the flavor]({{< ref "core/guides/android/releasing#new-flavor-release" >}}).

Reply:
Nice done!